docs/resource-naming/README.md

```diff
@@ -0,0 +1,218 @@
+
+# Configuring the NVIDIA Device Plugin to report Different GPU Models with Volcano Capacity Scheduling
+
+Volcano v1.9.0 introduces capacity scheduling capabilities, allowing you to configure different quotas for various GPU types (essential in production environments). For example:
+
+```yaml
+apiVersion: scheduling.volcano.sh/v1beta1
+kind: Queue
+metadata:
+  name: queue1
+spec:
+  reclaimable: true
+  deserved: # set the deserved field.
+    cpu: 2
+    memory: 8Gi
+    nvidia.com/t4: 40
+    nvidia.com/a100: 20
+```
+
+However, the default Nvidia Device Plugin reports GPU resources as `nvidia.com/gpu`, which does not support reporting different GPU models as shown in the example.
+
+This guide walks you through configuring the NVIDIA Device Plugin to report different GPU models and integrating it with Volcano's capacity scheduling.
+
+## Install a Customized Device Plugin
+
+In this section, we will cover how to install a customized Device Plugin using Helm. Instructions for installing `helm` can be found [here](https://helm.sh/docs/intro/install/). For prerequisites to install the Device Plugin, refer to the [documentation](../../README.md#prerequisites) for detailed instructions. 
+
+If you are using the GPU Operator to manage all GPU-related components, there are two options. One option is to disable the GPU Operator's management of the Device Plugin and follow the steps in this section. The other option is to modify the GPU Operator configuration accordingly.
+
+### 1.1 Install a Custom Device Plugin
+
+Begin by setting up the plugin's `helm` repository and updating it as follows:
+
+```shell
+helm repo add nvdp https://nvidia.github.io/k8s-device-plugin
+helm repo update
+```
+
+Then verify that the modified release (`v0.16.1`) of the plugin is available:
+
+```shell
+$ helm search repo nvdp --version 0.16.1
+NAME                           CHART VERSION  APP VERSION    DESCRIPTION
+nvdp/nvidia-device-plugin      0.16.1     0.16.1        A Helm chart for ...
+```
+
+Next, prepare a ConfigMap file, assuming it's named `config.yaml`. A typical configuration is as follows:
+
+```yaml
+version: v1
+flags:
+  migStrategy: "none"
+  failOnInitError: true
+  nvidiaDriverRoot: "/"
+  plugin:
+    passDeviceSpecs: false
+    deviceListStrategy: "envvar"
+    deviceIDStrategy: "uuid"
+```
+
+Once this repo is updated and the ConfigMap is prepared, you can begin installing packages from it to deploy the `nvidia-device-plugin` helm chart.
```

Comment:
It's better to emphasize that user should replace device plugin images released from volcano.